bin/clang/astVisitorGlobalVars.cc

```diff
@@ -86,13 +98,15 @@ SkeletonASTVisitor::setupGlobalReplacement(VarDecl *D, const std::string& namePr
     var.typeStr = getCleanName(GetAsString(D->getType()));
     var.retType = var.typeStr + "*";
   } else if (D->getType().getTypePtr()->isFunctionPointerType()){
-    bool isTypeDef = isa<TypedefType>(D->getType().getTypePtr());
+    const Type* baseType = getPointerBaseType(D);
+    bool isTypeDef = isa<TypedefType>(D->getType().getTypePtr()) ||
+                     isa<TypedefType>(baseType);
```

Comment:
@jjwilke, it looks like you might be able to use getAs here, but I'm not 100% sure about that.